src/ecl/eclWatchTree.ts

```diff
@@ -371,13 +375,18 @@ export class ECLWUNode extends Item<ECLWatchTree> {
     }
 
     getDescription(): string {
+        let primary = this._wu.Wuid;
         const extras: string[] = [];
+        if (!this._wu.isComplete() || this._wu.isDeleted()) extras.push(this._wu.State);
+        if (!this._tree._myWorkunits && this._wu.Owner) extras.push(this._wu.Owner);
         if (this._wu.Jobname) {
+            primary = this._wu.Jobname;
             extras.push(this._wu.Wuid);
         }
-        if (!this._tree._myWorkunits && this._wu.Owner) extras.push(this._wu.Owner);
-        if (!this._wu.isComplete() || this._wu.isDeleted()) extras.push(this._wu.State);
-        return extras.join(" ");
+        if (this._wu.Protected) {
+            extras.push("Protected");
+        }
+        return extras.length ? `${primary} (${extras.join(", ")})` : primary;
     }
```

Comment:
Wuid can end up being displayed twice (label and description) - also now its been moved to the description the parenthesis are not needed?, how about:
```ts
    getDescription(): string {
        const extras: string[] = [];
        if (this._wu.Jobname) extras.push(this._wu.Wuid);
        if (this._wu.Protected) extras.push("Protected");
        if (!this._wu.isComplete() || this._wu.isDeleted()) extras.push(this._wu.State);
        if (this._wu.Owner) extras.push(this._wu.Owner);
        if (this._wu.Cluster) extras.push(this._wu.Cluster);
        return extras.join(", ");
    }
```